### ISSUE TITLE
Android: orientationListener fires only after calling lockScreenOrientation

### ISSUE THREAD

**Issue by JanKaifer** [wonday/react-native-orientation-locker#162]
Basically orientationChangeListener gets fired only when changing screens (because each screen sets its preferred orientation).

I was able to do this workaround:
```
  const {width, height} = useWindowDimensions();
  useEffect(() => {
    nativeStore.lockScreenOrientation(orientation);
  }, [orientation, width, height]);
```
(I changed dependencies from `[orientation]` to `[orientation, width, height]`)

Which works quite well.

**Comment by JanKaifer** [comment]
Could duplicate of #160
